src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,209 @@
+            deleteStatement.setLong(1, id);
+            return deleteStatement.executeUpdate() > 0;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't delete car with id: " + id, e);
+        }
+    }
+
+    private Car createCarObject(ResultSet resultSet) throws SQLException {
+        Car car = new Car();
+        Long carId = resultSet.getObject("id", Long.class);
+        String carModel = resultSet.getString("model");
+        car.setId(carId);
+        car.setModel(carModel);
+
+        Manufacturer manufacturer = new Manufacturer();
+        Long manufacturerId = resultSet.getObject("manufacturer_id", Long.class);
+        String manufacturerName = resultSet.getString("name");
+        String manufacturerCountry = resultSet.getString("country");
+        manufacturer.setId(manufacturerId);
+        manufacturer.setName(manufacturerName);
+        manufacturer.setCountry(manufacturerCountry);
+        return car;
+    }
+
+    private List<Driver> getDriversByCarId(Long id) {
+        List<Driver> driversList = new ArrayList<>();
+        String request = "SELECT * FROM drivers d JOIN cars_drivers cd "
+                + "ON d.id = cd.driver_id WHERE cd.car_id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement getDriversStatement = connection.prepareStatement(request)) {
+            getDriversStatement.setLong(1, id);
+            ResultSet resultSet = getDriversStatement.executeQuery();
+            while (resultSet.next()) {
+                driversList.add(createDriverObject(resultSet));
+            }
+            return driversList;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Cant get driver by car id: " + id, e);
+        }
+    }
+
+    private Driver createDriverObject(ResultSet resultSet) throws SQLException {
+        Driver driver = new Driver();
+        Long driverId = resultSet.getObject("id", Long.class);
+        String driverName = resultSet.getString("name");
+        String driverLicenseNumber = resultSet.getString("license_number");
+        driver.setId(driverId);
+        driver.setName(driverName);
+        driver.setLicenseNumber(driverLicenseNumber);
+        return driver;
+    }
+
+    private void deleteOldRelations(Long id) {
+        String request = "DELETE FROM cars_drivers WHERE car_id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement deleteRelationsStatement = connection.prepareStatement(request)) {
+            deleteRelationsStatement.setLong(1, id);
+            deleteRelationsStatement.executeUpdate();
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't delete relations "
+                    + "between car and driver by id: " + id, e);
+        }
+    }
+
+    private void createNewRelations(Car car, Driver driver) {
```

Comment:
```suggestion
    private void createNewRelations(Car car) {
```
we can get drivers from car field, no need to pass them to the method